BaseFilter/sections/foreign.txt

```diff
@@ -10650,6 +10650,11 @@ phimsexmoi.biz###text-2[style*="left: 0px; top: 0px;"]
 nhaccuatui.com###eventBannerLeft
 nhaccuatui.com###eventBannerTop
 nhaccuatui.com##div[class^="adv_"]
+truyenqqhot.com###ad_info_top
+truyenqqhot.com###ad_info
+truyenqqhot.com###ads_info
+truyenqqhot.com###ad_infos
+truyenqqhot.com##.text_ads
```

Comment:
It seems `truyenqqhot.com##div[id^="ad_"]` is enough.

Reply:
that filter looks dangerous. Recently they have an id called `ad_poster` which contains some announcements about the change

Reply:
What about `truyenqqhot.com##div[id^="ad_info"]`?

Reply:
Can you provide the URL?
Check this `truyenqqhot.com##div[id^="ad_"]:not([id*="_poster"])` or `truyenqqhot.com##div[id^="ad_"][id*="info"]`

Reply:
> What about `truyenqqhot.com##div[id^="ad_info"]`?

they have ad ids named `ads_info`

Reply:
I think I'll shorten the filter to
```
truyenqqhot.com##div[id^="ad_"]:not([id*="_poster"])
truyenqqhot.com###ads_info
truyenqqhot.com##.text_ads
```

Reply:
> Can you provide the URL? Check this `truyenqqhot.com##div[id^="ad_"]:not([id*="_poster"])` or `truyenqqhot.com##div[id^="ad_"][id*="info"]`

this is url https://truyenqqhot.com/truyen-tranh/sakurai-san-wa-kizuite-hoshii-8558-chap-10.html
they usually change the ids every week or even every day